entities.go

```diff
@@ -15,12 +15,14 @@ import (
 )
 
 type Network struct {
-	id       string
-	endpoint *net.UDPAddr
-	seed     []byte
-	pubkey   wgtypes.Key
-	route    string
-	ifname   string
+	id         string
+	endpoint   *net.UDPAddr
+	seed       []byte
+	pubkey     wgtypes.Key
+	route      string
+	ifname     string
+	gateway    string
```

Comment:
It's an interesting idea, but I'm not sure what it would bring. The parameters are coming from docker (ie. we don't have any control over the format, and if docker decides to change the format, the plugin would break). In the DB it will be stored as a string, because threre is not IP type for sqlite3.

Reply:
you are probably right, as it is now a net-specific data structure doesn't bring much value.
Let's go with the string 👍